src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverReportExtension.kt

```diff
@@ -388,6 +388,13 @@ public interface KoverHtmlReportConfig {
      */
     public var title: String?
 
+    /**
+     * Specify charset in HTML reports.
+     *
+     * If not specified, Charset.defaultCharset() is used instead.
```

Comment:
UTF-8 is default for [JaCoCo](https://www.jacoco.org/jacoco/trunk/doc/ant.html)

Reply:
ok! i'll change comment 😄